Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ButtonData.xml

```diff
@@ -2938,8 +2926,7 @@
         <EditorCategories value="Race:Terran"/>
     </CButton>
     <CButton id="AP_ValkyrieEnhancedClusterLaunchers">
-        <TintRacially value="1"/>
-        <Icon value="AP\Assets\Textures\btn-weapon-terran-hailstormmissiles.dds"/>
+        <Icon value="assets\textures\btn-upgrade-swann-multilockweaponsystem.dds"/>

```

Comment:
This shouldn't be same as Goliath's Multi-lock due to Brynhilds that use both of them (Muli-lock in Assault and Enhanced Cluster Launchers in Fighter)